bioconda_utils/bioconda_utils-conda_build_config.yaml

```diff
@@ -9,50 +9,46 @@
 # clear hard-coded default value for CONDA_BUILD_SYSROOT
 CONDA_BUILD_SYSROOT:
   - ""
+c_stdlib:
+  - sysroot                    # [linux]
+  - macosx_deployment_target   # [osx]
+c_stdlib_version:
+  - 2.17                       # [linux]
+  - 11.0                       # [osx and arm64]
+  - 10.13                      # [osx and x86_64]
 MACOSX_DEPLOYMENT_TARGET:      # [osx]
   - 11.0                       # [osx and arm64]
-  - 10.9                       # [osx and x86_64]
-
-pin_run_as_build:
-  htslib:
-    max_pin: x.x
-  bamtools:
-    max_pin: x.x.x
-  r-base:
-    max_pin: x.x
-    min_pin: x.x
+  - 10.13                      # [osx and x86_64]
```

Comment:
In the future we should also be able to drop `MACOSX_DEPLOYMENT_TARGET` and rely on the more uniform definition via `{{ stdlib("c") }}`.
Until we update our recipes to make use of it, we keep this defined here.

Reply:
So, the existing recipes can still work as-is, right? There is just the added benefit to using the new stdlib function?

Reply:
Exactly.
Depending on the decisions being made in `conda-forge`, we may want to add the `stdlib("c")` parts though sooner (but not needed for the next bulk rebuild) to ensure we get the right `sysroot_linux*` versions later
.
As is, the choice of the sysroot on Linux is done very messily by weighing down unwanted (newer) versions with https://github.com/conda-forge/linux-sysroot-feedstock/blob/2b16024e8907a07dc9430c3b4a18884bd679b65d/recipe/meta.yaml#L138-L140 but in the future those hacks may be removed.